lib/foreman_maintain/package_manager/yum.rb

```diff
@@ -102,6 +102,11 @@ def clean_cache
       yum_action('clean', 'all')
     end
 
+    def files_not_owned_by_package(directory)
+      find_cmd = "find #{directory} -exec /bin/sh -c 'rpm -qf {} &> /dev/null || echo {}' \\;"
```

Comment:
Is there any benefit to run 'rpm -qf {}' in new shell ?

Reply:
@jameerpathan111 request to test this one

Reply:
@upadhyeammit there is probably no other way the -exec could execute `rpm -qf {} &> /dev/null || echo {}` which in fact is a "bash script". Is there any better way to do that?

Reply:
This is the best possible way we have, dont have any good way for this one. So if @jameerpathan111 shows green flag for this one then this is ready to go !

Reply:
@upadhyeammit 
@lvrtelov is working on this and will comment on PR once she has tested it.